documentation/getting_started/configuring-addresses.adoc

```diff
@@ -1,14 +1,14 @@
 == Address types
 
 {ProductName} is configured with a set of addresses that you can use for
-messages. Currently, {ProductName} supports 4 different address types:
+messages. Currently, {ProductName} supports four different address types:
 
 * Brokered queues
 * Brokered topics (pub/sub)
 * Direct anycast addresses
 * Direct broadcast addresses
 
-See the <<address_model>> for details.
+For more details, see the xref:con-address-space-messaging[address space].
```

Comment:
"See the address space" reads a bit strange... Maybe "see the address space documentation" ?

Reply:
Agreed, fixed